src/store/getters/state-getters.ts

```diff
@@ -386,7 +386,8 @@ export const hasCorrectionDataChanged = (state: StateIF): boolean => {
       havePeopleAndRolesChanged(state) ||
       hasShareStructureChanged(state) ||
       areProvisionsRemoved(state) ||
-      haveNewResolutionDates(state)
+      haveNewResolutionDates(state) ||
+      hasContactInfoChanged(state)
```

Comment:
Remove this (and on line 401 below).

Change the business contact info does not create a correction filing.

Reply:
Removing it.